<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Compras</class>
 <widget class="QMainWindow" name="Compras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1332</width>
    <height>722</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Compras</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="botonEditar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>670</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Productos</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEliminarTodo">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Eliminar Todo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonAceptar">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>670</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Aceptar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEliminar">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>670</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonVentas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ventas</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonCompras">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 102, 0);</string>
    </property>
    <property name="text">
     <string>Compras</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonIngresar">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonInventario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Inventario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>ID Producto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>1341</width>
      <height>831</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEstadisitcas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Estadisticas</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaProductos">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>1071</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditProducto">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>80</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxDescripcion">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>40</y>
      <width>811</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditDescripcion">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>Descripcion</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCantidad">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>Cantidad</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditValorUnitario">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>120</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>Valor Total</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(57, 62, 70);
color: rgb(238, 238, 238);</string>
    </property>
    <property name="text">
     <string>Porcentaje</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPorcentaje">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>botonEditar</zorder>
   <zorder>label_3</zorder>
   <zorder>botonEliminarTodo</zorder>
   <zorder>botonAceptar</zorder>
   <zorder>botonEliminar</zorder>
   <zorder>botonVentas</zorder>
   <zorder>botonCompras</zorder>
   <zorder>botonIngresar</zorder>
   <zorder>botonInventario</zorder>
   <zorder>label_5</zorder>
   <zorder>botonEstadisitcas</zorder>
   <zorder>tablaProductos</zorder>
   <zorder>lineEditProducto</zorder>
   <zorder>comboBoxDescripcion</zorder>
   <zorder>label_6</zorder>
   <zorder>spinBoxCantidad</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>label_9</zorder>
   <zorder>lineEditPorcentaje</zorder>
   <zorder>lineEditValorUnitario</zorder>
   <zorder>lineEditDescripcion</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
